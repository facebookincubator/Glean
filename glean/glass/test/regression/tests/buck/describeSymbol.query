action: describeSymbol
args: "test/buck/t/buck_project/PATH/./foo"
