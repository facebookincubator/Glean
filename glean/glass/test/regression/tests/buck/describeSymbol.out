[
    "@generated",
    {
        "comments": [],
        "contains_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "extends_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "kind": 30,
        "language": 11,
        "location": {
            "filepath": "buck_project/TEST_TARGETS",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 12,
                "lineEnd": 18
            },
            "repository": "test"
        },
        "modifiers": [],
        "name": {
            "container": ".",
            "localName": "foo"
        },
        "pretty_comments": [],
        "repo_hash": "testhash",
        "sym": "test/buck/t/buck_project/PATH/./foo",
        "sym_location": {
            "filepath": "buck_project/TEST_TARGETS",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 12,
                "lineEnd": 18
            },
            "repository": "test"
        },
        "sym_other_locations": [],
        "type_xrefs": []
    }
]